cypress/e2e/charging_stations.cy.ts

```diff
@@ -0,0 +1,39 @@
+describe('Charging station actions', () => {
+  beforeEach(() => {
+    cy.clearAllCookies();
+    cy.clearAllLocalStorage();
+    cy.clearAllSessionStorage();
+
+    //TODO figure out why this is thrown!
+    const stub = cy.stub();
+    Cypress.on('uncaught:exception', (err, runnable) => {
+      if (err.message.includes('ResizeObserver')) {
+        stub();
+        return false;
+      }
+    });
+
+    cy.visit('http://localhost:5173/charging-stations');
+  });
+
+  it('Get Charging station actions', () => {
+    cy.getByData('citrine-os-icon').should(
+      'have.attr',
+      'src',
+      '/Citrine_OS_Logo.png',
+    );
+
+    cy.getByData('custom-action-dropdown-button').click();
+    cy.get('ul.ant-dropdown-menu[role="menu"] li[role="menuitem"]')
```

Comment:
can we put a data-attr on each button so that we dont have to select the 11th item? We have the items in that dropdown sorted by label so I can see the index being thrown off

Reply:
I wasn't able to, as this uses antd menue component passing in the menue items. So I couldn't find a spot that had some html where I could extend a `data-test` attribute. :( 
Happy to take a look together if you like.

Reply:
I tried to improve this now with looking for the menu item and did use the text. It will probably break with other languages though :(